examples/nemo/eg4/dawn_script.py

```diff
@@ -36,58 +36,55 @@
 '''Example wrapper to run PSyclone generated SIR code in DAWN.'''
 
 import textwrap
-import sys
-import argparse
-import ctypes
 import os.path
 from optparse import OptionParser
-from ctypes import *
-from config import __dawn_install_module__, __dawn_install_dawnclib__
+from ctypes import c_char_p, CDLL
+from config import __dawn_install_dawnclib__
 from dawn import *
 from dawn import sir_printer
 
-dawn = CDLL(__dawn_install_dawnclib__)
+DAWN = CDLL(__dawn_install_dawnclib__)
 
 # PSyclone code start
 # PSyclone code end
 
-parser = OptionParser()
-parser.add_option("-v", "--verbose",
+PARSER = OptionParser()
+PARSER.add_option("-v", "--verbose",
                   action="store_true", dest="verbose", default=False,
                   help="print the SIR")
 
-(options, args) = parser.parse_args()
+(OPTIONS, _) = PARSER.parse_args()
 
 
-## Print the SIR to stdout only in verbose mode
-if options.verbose:
+# Print the SIR to stdout only in verbose mode
+if OPTIONS.verbose:
     T = textwrap.TextWrapper(initial_indent=' '*1, width=120,
                              subsequent_indent=' '*1)
-    des = sir_printer.SIRPrinter()
+    DES = sir_printer.SIRPrinter()
 
-    #  des.visitGlobalVariables(hir.global_variables)
+    # DES.visit_global_variables(hir.global_variables)
```

Comment:
Can we remove this line or is it a reminder for something? If so, please add a comment.

Reply:
It was there so I left it in :-). It's now removed.